app/models/explainer.rb

```diff
@@ -121,7 +122,7 @@ def self.search_by_similarity(text, language, team_id)
     }
     response = Bot::Alegre.query_sync_with_params(params, "text")
     results = response['result'].to_a.sort_by{ |result| result['_score'] }
```

Comment:
Please replace this sorting by the same one we have for fact-checks (https://github.com/meedan/check-api/blob/7bac066b91f1dca8395af1c707520c980d44e7bf/app/models/concerns/smooch_search.rb#L105):

`sort_by{ |a| [a[1][:model] != Bot::Alegre::ELASTICSEARCH_MODEL ? 1 : 0, a[1][:score]] }`

Or, add this as an action item to CV2-5617.

Reply:
will add it as action item to CV2-5617

Reply:
You can click on "Resolve conversation" once you do so.

Reply:
Added as subtask as I am going to merge CV2-5617 [https://meedan.atlassian.net/browse/CV2-5779](https://meedan.atlassian.net/browse/CV2-5779)